js/reference.js

```diff
@@ -850,6 +850,19 @@ var ERMrest = (function(module) {
                     var col = this._ref.columns[i];
                     this._values[i] = col.formatvalue(this._data[col.name], {context:this._ref._context});
                 }
+
+                /*
+                 * use this variable to avoid using computed formatted values in other columns while templating 
+                 */
+                var formattedValues = [];
```

Comment:
The `pre_format` will take place in the `formatPresentation` function in [core.js](https://github.com/informatics-isi-edu/ermrestjs/pull/163/files#diff-d85c7ecdc144ac4aec3ee7a0b24c3f9fR1287).